code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -886,11 +897,63 @@ var/list/datum/absorbed_dna/hivemind_bank = list()
 	shield.creator = M
 	M.put_in_hands(shield)
 	playsound(loc, 'sound/effects/blobattack.ogg', 30, 1)
-	src.visible_message("<span class='warning'>The end of [M]\'s hand inflates rapidly, forming a huge shield-like mass!</span>",
+	src.visible_message("<span class='danger'>The end of [M]\'s hand inflates rapidly, forming a huge shield-like mass!</span>",
 							"<span class='warning'>We inflate our hand into a robust shield.</span>",
 							"<span class='warning'>You hear organic matter ripping and tearing!</span>")
 	gibs(src.loc)
 
+/mob/proc/horror_form()
+	set category = "Changeling"
+	set name = "Horror Form (40)"
+	set desc = "Tear apart your human disguise, revealing your true form."
+
+	var/datum/changeling/changeling = changeling_power(40,0,0)
+	if(!changeling)	return 0
+	src.mind.changeling.chem_charges -= 40
+
+	var/mob/living/M = src
+
+	M.visible_message("<span class='danger'>[src] writhes and contorts, their body expanding to inhuman proportions!</span>")
+	if(!do_after(src,60))
+		M.visible_message("<span class='warning'>[src]'s transformation abruptly reverts itself!</span>")
+		return 0
+
+	M.visible_message("<span class='danger'>[src] grows into an abomination and lets out an awful scream!</span>")
+	playsound(loc, 'sound/effects/greaterling.ogg', 100, 1)
+
+	var/mob/living/simple_animal/hostile/true_changeling/ling = new (get_turf(M))
+
+	if(istype(M,/mob/living/carbon/human))
+		for(var/obj/item/I in M.contents)
+			if(istype(I,/obj/item/organ))
+				continue
+			M.drop_from_inventory(I)
+
+	if(M.mind)
+		M.mind.transfer_to(ling)
+	else
+		ling.key = M.key
+	var/atom/movable/overlay/effect = new /atom/movable/overlay(get_turf(M))
+	effect.density = 0
+	effect.anchored = 1
+	effect.icon = 'icons/effects/effects.dmi'
+	effect.layer = 3
+	flick("summoning",effect)
+	QDEL_IN(effect, 10)
+	M.forceMove(ling) //move inside the new dude to hide him.
+	M.status_flags |= GODMODE //dont want him to die or breathe or do ANYTHING
+	addtimer(CALLBACK(src, .proc/revert_horror_form,ling), 10 MINUTES)
+
+/mob/proc/revert_horror_form(var/mob/living/ling)

```

Comment:
This assumes ling is still present.

What happens if it's not? (Well, it'll runtime for one.)